### ISSUE TITLE
Librera FD 8.9.5 fdroid crash

### ISSUE THREAD

**Issue by foss-** [foobnix/LibreraReader#1107]
Librera Reader: 8.9.5-frdoid
PixelOS 13.0

Since updating to 8.9.5 Librera Reader is no longer opening. Android shows dialog `Librerra FD keeps stopping`. This was not a problem with version 8.8.107-fdroid.

Another user in fdroid chat reported this problem is not happening for github version, only fdroid build.

Related to MuPdf new version it seems. Crash log (1 year):

https://bin.disroot.org/?ee387c061431f719#8sWfrwtam5pX2pK1b4bj4Ma6cRnLgxcJkQbSEj2CwbML

**Comment by licaon-kter** [comment]
I've updated the recipe, needed Jav17, but kept mupdf 1.21.

The error is about mupdf so odd:
```
java.lang.UnsatisfiedLinkError: No implementation found for int org.ebookdroid.droids.mupdf.codec.MuPdfDocument.getMupdfVersion() (tried Java_org_ebookdroid_droids_mupdf_codec_MuPdfDocument_getMupdfVersion and Java_org_ebookdroid_droids_mupdf_codec_MuPdfDocument_getMupdfVersion__)
	at org.ebookdroid.droids.mupdf.codec.MuPdfDocument.getMupdfVersion(Native Method)
	at com.foobnix.pdf.info.AppsConfig.<clinit>(AppsConfig.java:51)
	at com.foobnix.pdf.info.AppsConfig.loadEngine(Unknown Source:0)
	at com.foobnix.LibreraApp.onCreate(LibreraApp.java:54)
	at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1193)
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6738)
	at android.app.ActivityThread.access$1300(ActivityThread.java:238)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1914)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:223)
	at android.app.ActivityThread.main(ActivityThread.java:7705)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:952)
```

**Comment by 010110a** [comment]
I can confirm the issue on Samsung S9 Android 10 Librera FD 8.9.5. Downgraded to version 8.8.107-fdroid.
Thanx

**Comment by Norbu96** [comment]
I have the same issue. By downgrading the problem did not occur again.
Shiftphone 5me Android 8.1

**Comment by Captain-Rage** [comment]
Same here on a new Samsung S22. Installing an earlier version does not help. Perhaps clearing the app cache is needed.

**Comment by foss-** [comment]
Resolved with 8.9.12-fdroid, thanks.
